kola/tests/misc/verity.go

```diff
@@ -74,8 +73,16 @@ func VerityCorruption(c cluster.TestCluster) {
 	// skip unless we are actually using verity
 	skipUnlessVerity(c, m)
 
+	out, err := c.SSH(m, "findmnt -n -o FSTYPE /usr")
```

Comment:
We could first check if the commands return an error then check the content of the output, something like:
```suggestion
	out, err := c.SSH(m, "findmnt -n -o FSTYPE /usr")
	if err != nil {
	    c.Fatalf("failed checking /usr filesystem: %s: %v", out, err)
	}
	
	if bytes.Equal(out, []byte("btrfs")) {
	    c.Skip("test does not support btrfs yet")
	}
	
	
```

Reply:
Yes, thanks, that's a bit clearer